include/psa/crypto_sizes.h

```diff
@@ -208,6 +208,18 @@
  * operations, and does not need to accept all key sizes up to the limit. */
 #define PSA_VENDOR_RSA_MAX_KEY_BITS 4096
 
+/* The minimum size of an RSA key on this implementation, in bits.
```

Comment:
This is the minimum size when generating, not when importing.

This makes the name misleading, especially given that there's also a `MAX` constant with a matching name that also applies when importing a key. I think the difference in behavior is justified: the minimum is to avoid accidental generation of an insecure key, which is not a concern when importing an existing key; the maximum is to avoid doing very slow or memory-intensive private-key computations. But then we should use a different name. I think `PSA_VENDOR_RSA_GENERATE_MIN_KEY_BITS` would be the natural name.

Reply:
Changed it to `PSA_VENDOR_RSA_GENERATE_MIN_KEY_BITS`